### PR TITLE
Don't add || in the display text

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: static/js/apps/eval_rig/util.ts [hunk 1]
```diff
@@ -37,11 +39,12 @@ export const processText = (text: string): string => {
...
       }
       let innerHtml = "";
-      innerHtml += `<span class="dc-stat">${dcStat || "&nbsp;&nbsp;"}</span>||`;
-      innerHtml += `<span class="llm-stat">${llmStat || "&nbsp;&nbsp;"}</span>`;
+      innerHtml += `<span class="dc-stat">${dcStat || longSpaces}</span>`;
+      innerHtml += `<span class="llm-stat">${llmStat || longSpaces}</span>`;
       return `<span class="annotation annotation-${callId}">${innerHtml}</span>`;
     }
   );
+  console.log(processedText);
```
Comment (chejennifer) [inline static/js/apps/eval_rig/util.ts:47]: remove console log?

File: static/js/apps/eval_rig/util.ts [hunk 2]
```diff
@@ -16,6 +16,8 @@
 
 const httpPattern = /https:\/\/[^\s]+/g;
 
+const longSpaces = "&nbsp;&nbsp;&nbsp;&nbsp;";
```
Comment (chejennifer) [inline static/js/apps/eval_rig/util.ts:19]: I see that long spaces used to be 2 spaces, did you mean to update to 4 spaces?

**Review by chejennifer — APPROVED** [review]
nice

File: static/js/apps/eval_rig/util.ts (hunk 2 above)
Comment (shifucun) [reply static/js/apps/eval_rig/util.ts:19]: Yes, purposely making this larger so it's not skipped.

File: static/js/apps/eval_rig/util.ts (hunk 1 above)
Comment (shifucun) [reply static/js/apps/eval_rig/util.ts:47]: Done

**Review by pradh — APPROVED** [review]
Thanks!

**Pull request merged**